cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -178,8 +203,12 @@ impl Node<Option<cst::Policy>> {
     }
 
     /// Convert `cst::Policy` to an AST `InlinePolicy`. (Will fail if the CST is for a template)
-    pub fn to_policy(&self, id: ast::PolicyID) -> Result<ast::StaticPolicy> {
-        let maybe_template = self.to_policy_template(id);
+    pub fn to_policy_helper(
```

Comment:
Is this part of the public API for ASTs? If the answer is yes, then I'm not sure about renaming the function. Consumers are going to break anyway due to the extra `allow_errors` argument. We could even go as far as to create new `to_policy_with_errors` instead of changing this, but that'd lead to duplicated code for `to_policy_template` which I don't think we want.

Reply:
The helper functions should not be public, I'll fix that.

I am also planning to add a feature flag so that none of this error functionality will work unless it is explicitly enabled